scripts/acquire_cryo.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Created on 12 August 2019
+
+@author: Anders Muskens
+
+Run a cryo acquisition
+
+You first need to run the Odemis backend with the SECOM or Delphi config. For
+instance, start odemis and close the graphical interface. Alternatively you can
+start just the backend with a command such as:
+odemisd --log-level 2 /usr/share/odemis/secom.odm.yaml
+
+The configuration used is the settings of the hardware just _before_ starting
+the script.
+"""
+
+from __future__ import division
+
+import argparse
+import logging
+import sys
+from odemis import dataio, model
+
+logging.getLogger().setLevel(logging.INFO)
+
+
+def acquire_confocal(scanner, det, res, zoom, dt):
+    """
+    Acquire from det with scanner
+    with para: res (resolution), zoom (zoom), dt (dwelltime)
+    """
+    max_res = scanner.resolution.range[1]
+    scale = [m / (r * zoom) for m, r in zip(max_res, res)]
+    scanner.scale.value = scale
+    scanner.resolution.value = res
+    scanner.dwellTime.value = dt
+
+    im = det.data.get()
+    if det.protection.value:
+        logging.warning("Protection activated")
+        det.protection.value = False
+
+    return im
+
+
+def acquire_sem(dt, scale):
+    ebeam = model.getComponent(role="e-beam")
+    sed = model.getComponent(role="se-detector")
+
+    ebeam.dwellTime.value = dt
+    ebeam.scale.value = scale
+    ebeam.resolution.value = (1, 1)
+    logging.info("Acquiring....")
+    im = sed.data.get()
+    logging.info(im[0][0])
+    return im
+
+
+def main(args):
+    """
+    Handles the command line arguments
+    args is the list of arguments passed
+    return (int): value to return to the OS as program exit code
+    """
+    # Get components
+    # stage = model.getComponent(role="scan-stage")
+    #scanner = model.getComponent(role="laser-mirror")
+    """
+
+    # parse arguments
+    parser = argparse.ArgumentParser(description="Run an acquisition "
+                                     "with Cryo hardware for testing")
```

Comment:
I don't quite understand what does this script (better than just `odemis-cli --acquire se-detector ...`), but in any case it should be in a separate PR from the extension of the SmarAct driver.